README.md

```diff
@@ -34,6 +34,27 @@ NOTE: On Windows, you can install it using [choco](https://chocolatey.org/instal
 No other packages or manual setup is required as these will be handled with user prompts after install.
 However, you may wish to install `rustup` with your OS package manager instead of following prompts to install via [rustup.rs](https://rustup.rs).
 
+## Configure rust-analyzer
+
+**rust-analyzer** settings can be stored in a JSON file located at `.ide-rust/rust-analyzer.json` relative to the project directory.
+
+### Example
+
+`.ide-rust/rust-analyzer.json`
```

Comment:
Only these paths are remaining. The rest is great. Thanks!

Reply:
Right good catch, thank you.